demo/demo_client.c

```diff
@@ -468,8 +475,15 @@ xqc_demo_cli_keylog_cb(const char *line, void *engine_user_data)
         return;
     }
 
-    write(ctx->keylog_fd, line, strlen(line));
-    write(ctx->keylog_fd, "\n", 1);
+    int write_len = write(ctx->keylog_fd, line, strlen(line));
+    if (write_len < 0) {
+        printf("write keys failed, errno: %d\n", errno);
+        return;
+    }
+    write_len = write(ctx->keylog_fd, "\n", 1);
```

Comment:
forgotten the definition line_break I must have, help to replace the "\n" character with it please.